app/assets/javascripts/rad_common/autocomplete-rails.js

```diff
@@ -126,16 +126,30 @@
           if(jQuery(e).data('excluded-ids')) {
               params['excluded_ids'] = jQuery(e).data('excluded-ids');
           }
+          const allowNewModel = jQuery(e).attr('data-allow-new-model');
           jQuery.getJSON( jQuery(e).attr('data-autocomplete'), params, function() {
+            if (!!allowNewModel) {
+              arguments[0].unshift({ id: '', label: `Create "${request.term}" ${allowNewModel}` });
```

Comment:
adds create text if allow new model, doesnt really do anything, just closes the dropdown